78_devspace_part_3/README.md

```diff
@@ -0,0 +1,166 @@
+# Devspace, Part 3
+
+In this lecture we get started with `DevSpace`, which is an open-source developer tool for Kubernetes that lets you develop and deploy cloud-native software. We are talked about possibilities that gives `DevSpace` and generated first devspace.yaml for devspace-example-python project. The main components of the deployment manifest are discussed.
```

Comment:
`In this DevSpace lecture, participants will explore running the "02-fastapi-tortoise" example project and an 
introduction to the Nix package manager.

Attendees will learn how to run the "02-fastapi-tortoise" example project, which is built using FastAPI and Tortoise ORM. FastAPI is a modern Python web framework known for its speed and ease of use, while Tortoise ORM provides an object-relational mapping for working with databases.

Nix package manager, a powerful tool used for managing software packages and system environments. Nix adopts a functional approach, treating packages as immutable units with unique identifiers, ensuring reproducibility and avoiding dependency conflicts. `